src/fmu/dataio/aggregation.py

```diff
@@ -230,32 +232,28 @@ def _generate_aggrd_metadata(
         template["fmu"]["context"]["stage"] = "iteration"
```

Comment:
Not directly related, perhaps, but this line could have some sort of explanation. I.e. something like "For now, we hard code the context for aggregated data to "iteration", based on current patterns in FMU. However, if we wish to aggregate in different contexts in the future, this will have to change."

(Technically and conceptually, "aggregations" are not bound to the iteration context - but conventionally, currently, they are.)